colin/core/config/config.py

```diff
@@ -125,6 +125,24 @@ def get_checks_path():
     return os.path.abspath(os.path.join(__file__, rel_path))
 
 
+def get_config_file(config):
+    """
+    Get the config file from name or path
+
+    :param config: str (name or path)
+    :return: str
+    """
+    if os.path.exists(config) and os.path.isfile(config):
+        return config
+
+    config_directory = get_config_directory()
+    config_file = os.path.join(config_directory, config + JSON)
```

Comment:
if `config` is a directory then it will fail, isn't it?

Reply:
It will fail on the next check:
```
    if os.path.exists(config_file) and os.path.isfile(config_file):
        return config_file
    raise ColinConfigException("Config '{}' cannot be found.".format(config))
```